src/components/TournamentMarker.jsx

```diff
@@ -45,9 +45,9 @@ const TournamentMarker = memo(({ tournament }) => {
         <ReactLeaflet.Marker ref={markerRef} key={tournament.url} position={tournament.location} icon={defaultIcon}>
             <ReactLeaflet.Popup>
                 <h2>
-                    <a href={tournament['url']} target="_blank" rel="noreferrer">{tournament['name']}</a>
+                    <a href={tournament.url} target="_blank" rel="noreferrer">{tournament.name}</a>
```

Comment:
Just out of curiosity, do we get any extra undefined-checking here or anything by switching how we access the properties? This certainly looks nicer and it gets a thumbs-up from me regardless, I'm just wondering if it's pure aesthetic or not.

Reply:
Purely aesthetics in this case.

I use dot notation when I know what I'm digging for, but if I'm iterating over an objects list of fields I'll use bracket notation and call it obj[key] or something and work with that.

Reply:
Makes sense, I dig it.